lib/trento/application/integration/discovery/payloads/saptune_discovery_payload.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Trento.Integration.Discovery.SaptuneDiscoveryPayload do
+  @required_fields nil
+
+  use Trento.Type
+
+  deftype do
+    field :package_version, :string
+    field :configured_version, :string
+    field :tuning_state, :string
+    field :services, :map
+    field :notes_enabled_by_solution, {:array, :map}
+    field :notes_applied_by_solution, {:array, :map}
+    field :notes_enabled_additionally, {:array, :string}
+    field :solution_enabled, {:array, :map}
```

Comment:
I think `solution_enabled` should be an array of strings:
```
"Solution enabled": [
    "NETWEAVER"
],
```

Reply:
done, thx for spotting